### ISSUE TITLE
Ammo still spawns with AutoSpawnable false

### ISSUE THREAD

**Issue by TheSuspect2306** [TTT-2/TTT2#948]
For example there is a random weapon with 3 Ammo and the weapon has Ammo that should not spawn it still spawns even with AutoSpawnable to false and spawnType to nil

**Comment by ZenBre4ker** [comment]
As ammo is random are you sure there just doesnt spawn other ammo types you didnt disable?

**Comment by TheSuspect2306** [comment]
every ammo type spawns regardless of AutoSpawnable false or true, i think @TimGoll could explain more to this bug

**Comment by ZenBre4ker** [comment]
Yeah thats because of our system. We dont check for ammunition as item, but only spawn their entities.
So this is accepted. We could implement a check for the ammo itself and port the main ttt-ammo to the equipment-editor to enable deactivation.

**Comment by TimGoll** [comment]
I think you two are talking about different things. Right now, the random ammo entity only spawns random ammo that ha the flag set to true. However this flag is ignored if you set an ammo count to a weapon spawn. So I'm with @TheSuspect2306 here

**Comment by mognify** [comment]
i might be jumping the gun, but i dont see anywhere in the code that changes any of the AutoSpawnable variables in any of the item/weapons, seems like they're all hardcoded to be AutoSpawnable = true with nothing to change them
